HIP/hip-336.md

```diff
@@ -192,10 +203,19 @@ The total number of approvals contained within the transaction body cannot excee
 
 The `amount` must be specified in terms of the `decimals` value for the token. If `amount` is 0 the transaction will remove the `spender`'s allowance from the owner's account, thereby freeing an allowance for future use.
 
+A single NFT serial number can only be granted to one spender at a time. If an approval assigns a previously approved NFT serial number to a new user, the old user will have their approval removed. For example, if the following scenario were to occur:
+
+1. *userA* approves *userB* to use NFT *T* with serial *S*
+2. *userA* approves *userC* to use NFT *T* with serial *S*
+
+the end result will be that *userC* will hold an allowance to NFT *T* serial *S* while *userB* will have their allowance for NFT *T* serial *S* removed.
+
 In order to grant the `spender` access to *all* of the owner's instances of a particular NFT, the `approvedForAll` field in the `NftAllowance` message should be set to `true`.
 
 If the `spender` has already been granted `approvedForAll` for an owner's NFT and another transaction granting `approvedForAll` is sent for the same NFT, the transaction will be accepted and fees charged but it will effectively be a no-op. Similarly if multiple transactions to remove `approvedForAll` on the same NFT are received the effect will be idempotent.
 
+If the `spender` has been granted `approvedForAll` for an owner's NFT, the `spender` will in turn be permitted to approve an NFT serial number be spent by yet another spender. In other words, an `approvedForAll` spender effectively has the same privileges as the owner of the NFT. This behavior is consistent with the ERC721 implementation and is included here for backward compatibility of Ethereum smart contracts being ported over to Hedera.
```

Comment:
Should there be explicit sentence to state `approveForAll` can't be granted by a spender with `approvedforAll` ?

Reply:
This makes enforcing Signature requirements a bit weird. Additional flag to denote this behavior in the allowance txn would help.